public/scripts/experiments/stevens.js

```diff
@@ -3,10 +3,19 @@ class Stevens {
   /**
    * Initializes a Stevens experiment object. 
    *
-   * @param  condition_name {string}    Name of condition (i.e foundational)
+   * @param  range          {string}    Range type (foundational or design)
+   * @param  condition_name {string}    Name of condition (i.e distractor_rainbow)
    * @param  graph_type     {string}    Name of graph_type
    */
-  constructor(condition_name, graph_type) {
+  constructor(range, condition_name, graph_type) {
```

Comment:
this constructor looks like the jnd constructor, probably another candidate for squashing the 2 classes together (for another refactor! don't do it in this PR)

Reply:
Like we talked about earlier, Stevens and JND are methodologically different, so not sure if we can merge the 2 classes together...